.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Good job adding node_modules to gitignore file. But let's add test.md to your gitignore file as well as requested on project requirements.
<img width="1077" alt="Screenshot 2023-02-15 at 12 25 59 AM" src="https://user-images.githubusercontent.com/25887253/218866952-03b1d74b-8a78-4ded-933d-01487a7ae5e1.png">

Since you have already uploaded/pushed to Github, you need to delete the file directly from Github after gitignoring from local repository 👍